### DISCUSSION TITLE
Welcome to Create Go App CLI Discussions!

### DISCUSSION THREAD

**Discussion by koddr** [create-go-app/cli#43] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ArunNKutty** [comment]
@koddr  This is an amazing tool that you have created . I was using it and maybe had suggestions or questions . I am trying to build a large microservices project based on a monorepo pattern where each monorepo is a docker container. The platform code is something that can be common for us . Should we use the same structure or would you advise a change in the structure

**Reply by koddr** [reply]
Hi! Thanks for the feedback, it's always nice to make other developers handy tools.

About your project. If you can elaborate (or show) your monorep, I can write my opinion in more detail.
